modules/accessing-inference-endpoint-for-model-deployed-on-single-model-serving-platform.adoc

```diff
@@ -71,7 +85,7 @@ https://<inference_endpoint_url>:443/api/v1/task/server-streaming-text-generatio
 ----
 grpcurl -proto text-generation-inference/proto/generation.proto -d \
 '{"requests": [{"text":"<text>"}]}' \
--H 'mm-model-id: <model_name>' -insecure <inference_endpoint_url>:443 fmaas.GenerationService/Generate \
+-insecure <inference_endpoint_url>:443 fmaas.GenerationService/Generate \
```

Comment:
Just curious why this changed?

Reply:
This is not required when querying tgis standalone

Reply:
This is not required when querying tgis standalone